core/src/main/java/org/apache/accumulo/core/util/ThreadPools.java

```diff
@@ -0,0 +1,368 @@
+    public <T> T invokeAny(Collection<? extends Callable<T>> tasks)
+        throws InterruptedException, ExecutionException {
+      return super.invokeAny(wrapCollection(tasks));
+    }
+
+    @Override
+    public <T> T invokeAny(Collection<? extends Callable<T>> tasks, long timeout, TimeUnit unit)
+        throws InterruptedException, ExecutionException, TimeoutException {
+      return super.invokeAny(wrapCollection(tasks), timeout, unit);
+    }
+
+    @Override
+    public void shutdown() {
+      super.shutdown();
+    }
+
+    @Override
+    public List<Runnable> shutdownNow() {
+      return super.shutdownNow();
+    }
+
+  }
+
+  /**
+   * ThreadPoolExecutor that traces executed tasks.
+   */
+  public static class TracingThreadPoolExecutor extends ThreadPoolExecutor {
+
+    private TracingThreadPoolExecutor(int corePoolSize, int maximumPoolSize, long keepAliveTime,
+        TimeUnit unit, BlockingQueue<Runnable> workQueue, ThreadFactory threadFactory) {
+      super(corePoolSize, maximumPoolSize, keepAliveTime, unit, workQueue, threadFactory);
+    }
+
+    @Override
+    public void execute(Runnable command) {
+      super.execute(new TraceRunnable(command));
+    }
+
+    @Override
+    public <T> Future<T> submit(Callable<T> task) {
+      return super.submit(new TraceCallable<T>(task));
+    }
+
+    @Override
+    public <T> Future<T> submit(Runnable task, T result) {
+      return super.submit(new TraceRunnable(task), result);
+    }
+
+    @Override
+    public Future<?> submit(Runnable task) {
+      return super.submit(new TraceRunnable(task));
+    }
+
+    private <T> Collection<? extends Callable<T>>
+        wrapCollection(Collection<? extends Callable<T>> tasks) {
+      List<Callable<T>> result = new ArrayList<Callable<T>>(tasks.size());
+      tasks.forEach(t -> result.add(new TraceCallable<T>(t)));
+      return result;
+    }
+
+    @Override
+    public <T> List<Future<T>> invokeAll(Collection<? extends Callable<T>> tasks)
+        throws InterruptedException {
+      return super.invokeAll(wrapCollection(tasks));
+    }
+
+    @Override
+    public <T> List<Future<T>> invokeAll(Collection<? extends Callable<T>> tasks, long timeout,
+        TimeUnit unit) throws InterruptedException {
+      return super.invokeAll(wrapCollection(tasks), timeout, unit);
+    }
+
+    @Override
+    public <T> T invokeAny(Collection<? extends Callable<T>> tasks)
+        throws InterruptedException, ExecutionException {
+      return super.invokeAny(wrapCollection(tasks));
+    }
+
+    @Override
+    public <T> T invokeAny(Collection<? extends Callable<T>> tasks, long timeout, TimeUnit unit)
+        throws InterruptedException, ExecutionException, TimeoutException {
+      return super.invokeAny(wrapCollection(tasks), timeout, unit);
+    }
+
+    @Override
+    public void shutdown() {
+      super.shutdown();
+    }
+
+    @Override
+    public List<Runnable> shutdownNow() {
+      return super.shutdownNow();
+    }
```

Comment:
There's a few of these methods that don't do anything other than call the superclass implementation. These overriding methods can be removed, so the caller of these methods can call the superclass' implementation directly.

Reply:
Yeah, these were leftovers from an earlier implementation idea.